config/tests/scenario_config_default.csv

```diff
@@ -0,0 +1,2 @@
+title;start;slurmConfig
+default;1;--qos=standby --nodes=1 --tasks-per-node=12 --wait
```

Comment:
if I understand correctly, [based on this document](https://slurm.schedmd.com/slurm.conf.html#OPT_WaitTime) `--wait`, it means that `start.R` waits until the run is ready? If I understand it correctly, this option is limited to 18 hours, 20 minutes anyway, so you may also set `--time 1092` which may increase the chances of the run starting early.

Reply:
slurm is confusing, this is not `srun --wait=x`, this is [`sbatch --wait`](https://slurm.schedmd.com/sbatch.html#OPT_wait), which means it just waits for the job to be finished, then exits. So start.R waits until the run is finished, whatever it takes. It is needed because otherwise start.R just submits the job and then exits successfully.